services/minespace-web/src/styles/components/ProjectSummaries.scss

```diff
@@ -0,0 +1,3 @@
+.fixed-header {
+  // position: fixed;
```

Comment:
is this css needed anymore?

Reply:
no, this file should be deleted. I'll delete

Reply:
file deleted!